src/components/Header/SearchBar.tsx

```diff
@@ -54,6 +54,11 @@ export default function SearchBar({
       'tags'
     ])
     router.push(`${url}&text=${urlEncodedValue}`)
+
+    // clear input after search
```

Comment:
Understood. To me, it just made more sense, but it's just my opinion. Another thing that I find quite weird is the behaviour of the 'X'. When I click on it, I'd expect the result to clear up too, instead it stays the same. Do you think it makes sense to open an issue about it?

Reply:
dang, I could think of so much with the search but just pragmatically speaking we shouldn't bother right now and save this all for #666.

Reply:
we don't control the x right now, it's your native browser input dismissal. But you're right that something should happen when that x is hit, but not sure an empty search screen is it cause search can be invoked from anywhere could also make the case to restore the view the user invoked the search from so we quickly get complex with current implementation.

I think if we move/extend/whatever the searchbox on active search result page, then this would all be more clear. But again, more important that crucial functionality works before we start optimizing things like this.

Reply:
ok great 👍🏽 sounds good